pages/EC-JBoss_help.xml

```diff
@@ -1,1134 +1,991 @@
-	</ul>
-
-	<h2>EC-JBoss 2.1.2</h2>
-        <ul>
-          <li>Improved the plugin for better windows support. </li>
-          <li>Replaced the Username and Password fields with credentials in the CreateDatasource procedure for backward incompatibility.</li>
-        </ul>
-	<h2>EC-JBoss 2.1.1</h2>
-	<ul>
-	  <li>Cleaned up the code.</li>
-	  <li>Fixed the procedure descriptions and updated the documentation.</li>
-	</ul>
-        <h2>EC-JBoss 2.1.0 (February 24, 2015)</h2>
-		   <ul>
-            <li>Added the RunCustomCommand procedure to run user-specified commands on JBoss instances.</li>
-            <li>Added the StartServers procedure to start server groups.</li>
-            <li>Added the StopServers procedure to stop server groups.</li>
-            <li>Fixed the CreateDatasource procedure.</li>
-            <li>Fixed the DeleteDatasource procedure.</li>
-            <li>Fixed the DeployApp procedure.</li>
-            <li>Fixed the UndeployApp procedure</li>
-            <li>Fixed the EnableDeploy procedure.</li>
-            <li>Fixed the DisableDeploy procedure.</li>
-            <li>Fixed plugin errors on Linux ElectricFlow instances.</li>
-        </ul>
-
-				<h2>EC-JBoss 2.0.7 (December 2014)</h2>
-        <ul>
-            <li>Fixed a POST_PROCESSOR_ERROR in the CheckServerStatus procedure.</li>
-            <li>Fixed an error in the CheckDeployStatus procedure where the <i>Success Criteria</i> was set to "Page not found."</li>
-            <li>Fixed a bug in the DeployApp procedure where the "Runtime name for deployment" was specified.</li>
-        </ul>
-
-        <h2>EC-JBoss 2.0.6 (June 2012)</h2>
-		    <ul>
-			      <li>Added five new procedures to check the status of deployments, create datasources, delete datasources,
-						stop servers, and restart servers.</li>
-            <li>Fixed minor bugs.</li>
-		    </ul>
-
-        <h2>EC-JBoss 2.0.5 (April 2012)</h2>
-        <ul>
-            <li>Changed the procedure names in the step picker section.</li>
-        </ul>
-
-        <h2>EC-JBoss 2.0.4 (March 2012)</h2>
-        <ul>
-            <li>Did minor fixes.</li>
-        </ul>
-
-        <h2>EC-JBoss 2.0.3</h2>
-        <ul>
-            <li>Made improvements to the Help document.</li>
-        </ul>
-
-			  <h2>EC-JBoss 2.0.2</h2>
-        <ul>
-            <li>Upgraded the plugin to use the new XML parameter form.</li>
-            <li>Added a direct link to the new Help document.</li>
-        </ul>
-
-        <h2>EC-JBoss 2.0.1</h2>
-        <ul>
-            <li>Improved the XML parameter panels.</li>
-            <li>Reformatted the Help document.</li>
-        </ul>
-
-    </div>
-
-</body>
-
+    <head>
+        <meta name="generator" content="HTML Tidy for Windows (vers 14 February 2006), see www.w3.org" />
+        <meta content="text/html; charset=us-ascii" http-equiv="content-type" />
+        <title>ElectricFlow Plugin for @PLUGIN_KEY@</title>
```

Comment:
I guess here we will get "ElectricFlow Plugin for EC-JBoss". It's wrong, it should be ElectricFlow plugin for JBoss.